backend/db/utils.py

```diff
@@ -2,6 +2,8 @@
 
 todo's
   1. Some functions could be improved using `sql_query()` `params` arg to be less prone to sql injection.
+  2. Making 'Connection' optional: Can write a wrapper function and decorate all functions that need, where all it does
```

Comment:
### Unrelated idea 1: Making 'Connection' optional
Can write a wrapper function and decorate all such SQL based utility functions, where all it does is `con = con if con else get_db_connection()` and then pass that to inner function.